.gitignore

```diff
@@ -3,3 +3,4 @@
 bundle.Dockerfile
 RELEASE_BODY.md
 jsonnet/vendor
+config.yaml
```

Comment:
why is this required ? 🤔

Reply:
The idea behind this is to keep the config.yaml out of the base of the code if you were to have run a 3 node kind cluster. Looking at your last comment, this will not be necessary if we move this config.yaml to `hack/kind/config.yaml` 

```
cat <<EOF > config.yaml
kind: Cluster
apiVersion: kind.x-k8s.io/v1alpha4
nodes:
- role: control-plane
  image: kindest/node:v1.22.4
- role: worker
  image: kindest/node:v1.22.4
- role: worker
  image: kindest/node:v1.22.4
EOF
kind create cluster --config=config.yaml
```